draft-ietf-jsonpath-base.md

```diff
@@ -1235,6 +1239,7 @@ JSON:
 | Comparison | Result | Comment |
 |:--:|:--:|:--:|
 | `$.absent1 == $.absent2` | true | Empty nodelists |
+| `$.absent1 <= $.absent2` | true | `==` implies `<=` |
 | `$.absent1 == 'g'` | false | Empty nodelist |
 | `$.absent1 != $.absent2` | false | Empty nodelists |
 | `$.absent1 != 'g'` | true | Empty nodelist |
```

Comment:
Nit: In line 1243  `$.absent == 'g'`  would be cleaner. Same in line 1245.

Reply:
Yes, thank, fixed that in spite of it being somewhat out of scope of this PR.